src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -282,46 +282,55 @@ def generate_FDB_2_default_config(self, filename):
     device_b = json.load(
         open(os.path.join('testcases', 'testdata', 'device_b.json')))
 
+    # DPA database test harness configuration
+    dpa_database_config = {
+        'hostName': 'localhost',
+        'port': 8000,
+        'fileUrl': '/db_sync',
+        'filePath': os.path.join('testcases', 'testdata', 'fdb_2', 'FDB_2_Portal_DPAs.kml'),
+        'modifiedFilePath': os.path.join('testcases', 'testdata', 'fdb_2', 'modified_FDB_2_Portal_DPAs.kml')
+    }
+
     # Load grant requests
     grant_g1_a = json.load(
                 open(os.path.join('testcases', 'testdata', 'grant_0.json')))
-    # Set the grant frequency to overlap with the DPA 'puerto_rico_ver2_dpa_4'.
+    # Set the grant frequency to overlap with the DPA 'Pensacola' which is 3550-3700 MHz.
```

Comment:
DPA for Pensacola is 3500-3650, not including the last 50 MHz.

Reply:
fixed.

Reply:
Hmm, if so, that is an error. Pensacola is protected 3500-3700 but the 3650-3700 portion is covered by an exclusion zone, not a DPA (you can double-check with Masoud if you want).

Reply:
Sorry, actually I should have said: Pensacola is an ESC-monitored DPA and is therefore not applicable to this test case. Please pick a portal-controlled DPA instead.